src/sentry/plugins/sentry_mail/models.py

```diff
@@ -123,6 +125,27 @@ def get_send_to(self, project):
             logger.debug('Tried to send notification to invalid project: %r', project)
             return []
 
+        if event:
+            owners = ProjectOwnership.get_owners(project.id, event.data)
```

Comment:
See: https://github.com/getsentry/sentry/pull/7541

After this lands, you'll want to just do:

```python
owners, _ = ProjectOwnership.get_owners(project.id, event.data)
```

instead. We're not doing anything yet with the matcher (2nd return value) in here yet. That could be used if we wanted to say why they got the email or something in the future.